files/ko/web/css/_doublecolon_-webkit-scrollbar/index.md

```diff
@@ -91,32 +91,6 @@ Who thought scrollbars could be made weeeeird?
 
 ## 브라우저 호환성
 
-### `::-webkit-scrollbar`
-
-{{Compat}}
-
-### `::-webkit-scrollbar-button`
-
-{{Compat}}
-
-### `::-webkit-scrollbar-thumb`
-
-{{Compat}}
-
-### `::-webkit-scrollbar-track`
-
-{{Compat}}
-
-### `::-webkit-scrollbar-track-piece`
-
-{{Compat}}
-
-### `::-webkit-scrollbar-corner`
-
-{{Compat}}
-
-### `::-webkit-resizer`
-
```

Comment:
There are cases like this.. I'll have to be careful when writing regex in the future.

Reply:
All good, there's always some special case in these big refactoring projects that's unique to that one page/group of pages!